floss/main.py

```diff
@@ -99,6 +99,7 @@ def make_parser(argv):
         " 2. stack strings:   strings constructed on the stack at run-time\n"
         " 3. tight strings:   special form of stack strings, decoded on the stack\n"
         " 4. decoded strings: strings decoded in a function\n"
+        " 5. Go strings: strings extracted from Go binaries\n"
```

Comment:
let's list language specific strings separately as they are in a different category compared to the other ones listed here